.travis.yml

```diff
@@ -1,15 +1,16 @@
-language: node_js
-node_js:
-    - "0.10"
+language: php
+php:
+    - "5.6"
```

Comment:
`~5.5|~7.0` means 5.x starting 5.5.0 and 7.x starting 7.0.x, so it should mean 5.5, 5.6 and 7.0 at the moment.
Make sure you have an up-to-date composer, and if this is not working as intended, propose composer.json fixes that solves this. *(Aka 5.6 is supposed to be supported)*